CODEOWNERS

```diff
@@ -168,6 +170,7 @@
 /gallery/workbook/microsoft.azurepercept-accounts.json @microsoft/azure-percept-workbooks
 /gallery/workbook/microsoft.database-watcher.json @microsoft/sqlexternalmonitoring
 /gallery/apimanagement-service/** @microsoft/azure-api-management
+/gallery/private-mec/** @microsoft/azure-private-mec-workbooks
```

Comment:
unless we've creaetd a custom workbook "Type" for you (would require information from you about strings and help and support locations and things), this should just be "workbook" and your gallery file should be in the workbook folder, not a new type specific folder.

Reply:
Fixed in 03d2fbcdea790b7ca149a6b83d252cb94dea5350 - moved the file into the workbook folder and updated CODEOWNERS to match.